src/components/custom-field-input-text/custom-field-input-text.css

```diff
@@ -2,14 +2,18 @@
 @import '../../styles/colors-v2.css';
 @import '../../styles/spacing.css';
 @import '../../styles/typography.css';
+@import '../icon/icon.css';
 
 .input {
   border: 1px solid var(--mds-grey-12);
   border-radius: var(--spacing-x-small);
   box-sizing: border-box;
   color: var(--mds-grey-87);
   font-size: var(--mavenlink-type-base-size); /* stylelint-disable-line mds/typography */
-  padding: 7px var(--spacing-x-large) 7px var(--spacing-medium);
+  padding-top: 7px;
+  padding-right: calc((var(--numIcon, 0) * var(--iconSizeMedium)) + var(--spacing-medium)); /* stylelint-disable-line css-modules/css-variables */
```

Comment:
I think it might be worth re-factoring this to multiple classes that add more padding as we need it, which are added depending on the state of the input. With that approach, we wouldn't have more padding than we need for the state, and wouldn't have to do the math here in CSS.

Reply:
What are the advantages of not doing the math in CSS? We actually started with an approach like that and went to this >.<

Reply:
Really just keeping the CSS as simple as possible, I don't think there are any performance implications or anything. Might have a slightly negative effect on redraw? But really, just to #keeplogicoutofcss.

Reply:
If there is a nice way to pull `var(--spacing-medium)` into react-land, then I would be all for this. Is there some pattern I could use?

Since we define variables in css, I think it is nice to use those variables to do some simple math in css-land. Otherwise, I think we have to duplicate those variables in react.

Reply:
I think we process them into JS variables as part of the Webpack process, but I'm not sure exactly how that works or where they're accessible.